src/themes/brutalist.css

```diff
@@ -0,0 +1,809 @@
+      --background-hover: color-mix(in oklab, var(--background), transparent 20%);
+    }
+  }
+
+  wa-rating {
+    --symbol-size: var(--wa-font-size-l);
+    --symbol-color: color-mix(in oklab, var(--wa-color-base-20), transparent 50%);
+    --symbol-color-active: var(--wa-color-neutral-spot);
+  }
+
+  wa-alert {
+    --border-width: 0 0 0 var(--wa-panel-border-width);
+    --content-color: var(--wa-color-text-normal);
+    --padding: var(--wa-space-s) var(--wa-space-m);
+
+    &::part(icon) {
+      --padding: var(--wa-space-m);
+    }
+
+    &[variant='brand'] {
+      --icon-color: var(--wa-color-brand-text-on-fill);
+    }
+    &[variant='success'] {
+      --icon-color: var(--wa-color-success-text-on-fill);
+    }
+    &[variant='warning'] {
+      --icon-color: var(--wa-color-warning-text-on-fill);
+    }
+    &[variant='danger'] {
+      --icon-color: var(--wa-color-danger-text-on-fill);
+    }
+    &[variant='neutral'] {
+      --icon-color: var(--wa-color-neutral-text-on-fill);
+    }
+  }
+
+  wa-carousel {
+    --pagination-color-activated: var(--wa-color-blue-70);
+    --pagination-color-resting: var(--wa-color-neutral-fill-highlight);
+  }
+
+  wa-carousel::part(scroll-container) {
+    padding: var(--wa-shadow-offset-y-level-1) var(--wa-shadow-offset-x-level-1); /* make room for the box-shadow */
+  }
+
+  wa-carousel::part(pagination-item) {
+    border-radius: var(--wa-corners-base);
+  }
+
+  wa-carousel-item {
+    border: var(--wa-panel-border-width) var(--wa-panel-border-style) var(--wa-color-surface-border);
+    box-shadow: var(--wa-shadow-level-1);
+  }
+
+  wa-badge {
+    --border-color: var(--wa-color-surface-border);
+    --content-color: var(--wa-color-base-10);
+    text-transform: uppercase;
+
+    &::part(base) {
+      padding: var(--wa-space-xs) var(--wa-space-s);
+    }
+
+    &[variant='brand'] {
+      --background: var(--wa-color-blue-90);
+    }
+    &[variant='success'] {
+      --background: var(--wa-color-green-90);
+    }
+    &[variant='warning'] {
+      --background: var(--wa-color-yellow-90);
+    }
+    &[variant='danger'] {
+      --background: var(--wa-color-red-90);
+    }
+    &[variant='neutral'] {
+      --background: var(--wa-color-base-90);
+    }
+  }
+
+  wa-tag {
+    --border-width: 0 0 0 var(--wa-panel-border-width);
+
+    &[variant='brand'] {
+      --border-color: var(--wa-color-brand-border-subtle);
+    }
+    &[variant='success'] {
+      --border-color: var(--wa-color-success-border-subtle);
+    }
+    &[variant='warning'] {
+      --border-color: var(--wa-color-warning-border-subtle);
+    }
+    &[variant='danger'] {
+      --border-color: var(--wa-color-danger-border-subtle);
+    }
+    &[variant='neutral'] {
+      --border-color: var(--wa-color-neutral-border-subtle);
+    }
+  }
+
+  wa-tag::part(base) {
+    padding: var(--wa-space-m);
+  }
+
+  wa-input::part(form-control-label),
+  wa-select::part(form-control-label) {
+    font-size: var(--wa-font-size-s);
+  }
```

Comment:
```suggestion
  wa-input::part(form-control-label),
  wa-select::part(form-control-label),
  wa-radio-group::part(form-control-label) {
    font-size: var(--wa-font-size-s);
  }
```

Looks like this `wa-radio-group` fella escaped us. 👇 
<img width="554" alt="Screenshot 2024-02-14 at 10 47 09 AM" src="https://github.com/shoelace-style/webawesome/assets/126139086/fe740fc8-2cf9-4ddc-82d5-9bddbc24cc7e">

Reply:
it's like it's _shouting_ at us or something! thanks for catching that.